src/components/footer/index.tsx

```diff
@@ -0,0 +1,318 @@
+                          color="white"
+                          borderBottom="red"
+                        >
+                          <List>
+                            <ListItem>asdasd</ListItem>
+                          </List>
+                        </MobileOnlyAccordionSection>
+                      </MobileOnlyAccordion>
+                    </GridItem>
+                    <GridItem>
+                      <MobileOnlyAccordion allowMultiple={true}>
+                        <MobileOnlyAccordionSection
+                          title="Über uns "
+                          color="white"
+                        >
+                          <List>
+                            <ListItem>asdasd</ListItem>
+                            <ListItem>asdasd</ListItem>
+                            <ListItem>asdasd</ListItem>
+                            <ListItem>asdasd</ListItem>
+                          </List>
+                        </MobileOnlyAccordionSection>
+                        <MobileOnlyAccordionSection
+                          title="Werbung"
+                          color="white"
+                        >
+                          <List>
+                            <ListItem>asdasd</ListItem>
+                          </List>
+                        </MobileOnlyAccordionSection>
+                      </MobileOnlyAccordion>
+                    </GridItem>
+                    <GridItem>
+                      <MobileOnlyAccordion
+                        allowMultiple={true}
+                        allowToggle={true}
+                      >
+                        <MobileOnlyAccordionSection
+                          title="Für Medien"
+                          color="white"
+                        >
+                          <List>
+                            <ListItem>asdasd</ListItem>
+                            <ListItem>asdasd</ListItem>
+                            <ListItem>asdasd</ListItem>
+                            <ListItem>asdasd</ListItem>
+                          </List>
+                        </MobileOnlyAccordionSection>
+                        <MobileOnlyAccordionSection
+                          title="Rechtliches"
+                          color="white"
+                        >
+                          <List>
+                            <ListItem>asdasd</ListItem>
+                          </List>
+                        </MobileOnlyAccordionSection>
+                      </MobileOnlyAccordion>
+                    </GridItem>
+                    <GridItem>
+                      <Stack
+                        paddingY={{ base: '2xl', md: 'md' }}
+                        paddingX={{ base: 'md', md: 0 }}
+                        spacing="md"
+                      >
+                        <Text textStyle="heading5">Apps</Text>
+                        <Link href="#">
+                          <Image
+                            src={googleplay}
```

Comment:
I really hope not but do you need those logos to be translated?

Reply:
Nah don't think so the links might be